libs/models/race.ts

```diff
@@ -0,0 +1,24 @@
+import { PlayerWithLogs } from './player';
+
+export interface Race {
+  /** Unique race id */
+  id: string;
+
+  /** Race text which players are typing  */
+  text: string;
+
+  /** Race timeout timer which calculated by the server */
```

Comment:
specify the unit. (seconds ? milliseconds?)

Reply:
Specified